src/gasman.c

```diff
@@ -1805,82 +1799,6 @@ UInt CollectBags (
 
     /* * * * * * * * * * * * * * * sweep phase * * * * * * * * * * * * * * */
 
-#if 0
-    /* call freeing function for all dead bags                             */
-    if ( NrTabFreeFuncBags ) {
```

Comment:
They are used in libsing and NormalizInterface and soon more. And also HPC-GAP (though of course we don't use GASMAN there)